src/Shared/AppSettings.cs

```diff
@@ -11,10 +11,18 @@ public record CvPartnerConfig
     public string Token { get; set; }
 }
 
+public record BlobStorageConfig
+{
+    public Uri connectionString { get; set; }

```

Comment:
Vi burde bruke Managed Identity for å kontakte denne, men vi kan se på det senere. Burde legges inn i KeyVault eller AppConfig

Burde være stor C i ConnectionString, samme med proprertien under